txpipe/diagnostic_maps.py

```diff
@@ -390,6 +394,28 @@ def copy_attrs(tomo, name, out_name):
 
         meta_file.close()
 
+        # human readable version
+        yaml_out_name = self.get_output('tracer_metadata_yml')
+        metadata = {
+            'lens_density': lens_density.tolist(),
```

Comment:
do these come with units?

Reply:
I'm refactoring the mapping code in another branch - will add that there.